torch/fx/graph.py

```diff
@@ -74,13 +74,20 @@ def __init__(self):
     def nodes(self):
         return tuple(self._nodes)
 
-    def graph_copy(self, g : 'Graph'):
+    def graph_copy(self, g : 'Graph', val_map : Dict[Node, Node]) -> Optional[Node]:
         """
-        Append all nodes from graph `g` to this graph
+        Append all nodes from graph `g` to this graph. `val_map` should be a dictionary
+        that maps nodes in `g` to nodes in `self. `val_map` will be populated with more
+        items by this function. Returns the equivalent output value of `g` with
+        Nodes switched to refer to nodes in `self`.
         """
-        val_map : Dict[Node, Node] = {}
         for node in g._nodes:
+            if node.op == 'output':
+                rv = map_arg(node.args[0], lambda n: val_map[n])
+                assert isinstance(rv, Node)
```

Comment:
This assert doesn't have to be true. A graph can return a tuple.

Reply:
Return type should be `Optional[Argument]` which is just `Argument` since it already contains None.